app/src/main/java/org/openalpr/model/Results.java

```diff
@@ -16,11 +16,11 @@ public Results(Double epoch_time, Double processing_time_ms, ArrayList<Result> r
         this.results = results;
     }
 
-    public Double getEpoch_time() {
+    public Double getEpochTime() {
         return epoch_time;
     }
 
-    public Double getProcessing_time_ms() {
+    public Double getProcessingTimeMs() {
```

Comment:
https://github.com/SandroMachado/openalpr-android/blob/4aeb0dbddddb06bc993e28b95a0845d0a98e9c71/Sample/OpenALPRSample/app/src/main/java/com/sandro/openalprsample/MainActivity.java This method need to be updated here.